loki/transformations/tests/sources/projSccCuf/module/kernel.f90

```diff
@@ -42,14 +43,15 @@ PURE ELEMENTAL SUBROUTINE ELEMENTAL_DEVICE(x) ! elemental
       x = 0.0
     END SUBROUTINE ELEMENTAL_DEVICE
 
-    SUBROUTINE DEVICE(nlon, nz, jk_start, jl_start, jl_end, x)
-        INTEGER, INTENT(IN) :: jk_start, jl_start, jl_end
+    SUBROUTINE DEVICE(nlon, nz, jk_start, start, iend, x)
+        INTEGER, INTENT(IN) :: jk_start, start
+        INTEGER, INTENT(IN) :: iend
         INTEGER, INTENT(IN) :: nlon, nz
         REAL, INTENT(INOUT) :: x(nlon, nz)
         REAL    :: local_x(nlon, nz)
         INTEGER :: jk, jl
         DO jk = jk_start, nz
-            DO jl = jl_start, jl_end
+            DO jl = start, iend
```

Comment:
Hmm yes ... However, adding those as aliases doesn't solve the problem.
Introducing `bounds_aliases=('jl_start', 'jl_end')`  results in

```
    def __init__(self, name=None, index=None, bounds=None, size=None, aliases=None,
                 bounds_aliases=None, index_aliases=None):
        self.name = name
        self._index = index
        self._bounds = as_tuple(bounds)
        self._size = size
        self._aliases = as_tuple(aliases)
        self._index_aliases = as_tuple(index_aliases)
    
        if bounds_aliases:
            if len(bounds_aliases) != 2:
                raise RuntimeError(f'Start and end both needed for horizontal bounds aliases in {self.name}')
            if bounds_aliases[0].split('%')[0] != bounds_aliases[1].split('%')[0]:
>               raise RuntimeError(f'Inconsistent root name for horizontal bounds aliases in {self.name}')
E               RuntimeError: Inconsistent root name for horizontal bounds aliases in horizontal
```

Thus, more changes are required to make this work. Since I haven't seen this pattern yet I decided to change the variable names and adhere to the pattern we work with.
What do you think @reuterbal?

Reply:
I believe @awnawab has put some of this in, maybe he has seen cases where this would be relevant?
Otherwise fine to keep the naming consistent.

Reply:
The bounds aliases definitely need to be generalised to make it a more useful feature. When I put that feature in I didn't envisage a use case beyond using a derived-type container to store various dimension values, as is done in the dycore.

I am reasonably confident that the dycore use cases are captured well by tests, so please feel free to amend as needed.

Reply:
Therefore, I would suggest that I keep the rename for this transformation and we can further discuss whether we want/need to generalise this functionality (and make this a separate issue/PR)?

Reply:
Refer to #369